app/models/miq_product_feature.rb

```diff
@@ -18,8 +20,6 @@ class MiqProductFeature < ApplicationRecord
   validates_presence_of   :identifier
   validates_uniqueness_of :identifier
 
-  FIXTURE_PATH = Rails.root.join(*["db", "fixtures", table_name])
```

Comment:
haha, I didn't see this until now... I guess we were really worried about hardcoding "miq_product_features"... I can't believe I didn't see `table_name` before...